tests/integration/suite/test_pod_security_policies.py

```diff
@@ -162,6 +162,16 @@ def check_psptpb():
         return False
 
     wait_for(check_psptpb, lambda: "PSPTB project binding not found")
+
+    # check that an invalid podSecurityPolicyTemplateId causes 422
```

Comment:
The test looks good but lets move it out to it's own test. This one is already cramming a lot of functionality under the pretense of `delete`

Reply:
fixed. the test is moved to its own file.